swan-lake/learn/observing-ballerina-code.md

```diff
@@ -293,57 +286,53 @@ specification.](https://github.com/opentracing/specification/blob/master/semanti
 
 ### Configuring Advanced Tracing for Ballerina
 
-Tracing can be enabled in Ballerina with the `--b7a.observability.enabled=true` flag as mentioned in the [Observing a Ballerina Service](#observing-a-ballerina-service) section, as well as a configuration option. This section mainly focuses on the configuration options with the description and possible values.
+Tracing can be enabled in Ballerina with the few configurations as mentioned in the
+[Observing a Ballerina Service](#observing-a-ballerina-service) section.
```

Comment:
```suggestion
[Observing a Ballerina Service](#observing-a-ballerina-service).
```

Reply:
Fixed in 146f6434f70aee59d33f47a274e40ed2ac703d6a